src/src.pro

```diff
@@ -8,7 +8,7 @@ PLUGIN_IMPORT_PATH = io/thp/pyotherside
 
 TEMPLATE = lib
 CONFIG += qt plugin
-QT += qml quick
+QT += qml quick svg
```

Comment:
Hm, I wonder if we should make this optional, to reduce deployment size in cases where SVG support isn't needed?